src/modules/RemediationsModal/steps/selectPlaybook.js

```diff
@@ -91,6 +93,27 @@ const SelectPlaybook = (props) => {
     }
   });
 
+  //cannot use hook here, as it needs to be called in the OnChange function
+  const verifyName = (val) => {
+    setNewPlaybookName(val);
```

Comment:
I'm not following this 100% but it kinda looks like you're setting the new playbook name to the untrimmed version, but comparing the trimmed version to the existing plan names.  If you POST something to the api with leading or trailing spaces in the name you'll get a 400 back though, regardless of whether it's unique or not.

Reply:
added a trim to the api request